tests/test_datetime.py

```diff
@@ -11,10 +11,18 @@
     'value,dirty,expect_match',
     [
         pytest.param(datetime(2000, 1, 1), IsDatetime(approx=datetime(2000, 1, 1)), True, id='same'),
-        # Note: this requires the system timezone to be UTC
-        pytest.param(946684800, IsDatetime(approx=datetime(2000, 1, 1), unix_number=True), True, id='unix-int'),
-        # Note: this requires the system timezone to be UTC
-        pytest.param(946684800.123, IsDatetime(approx=datetime(2000, 1, 1), unix_number=True), True, id='unix-float'),
+        pytest.param(
+            946684800,
+            IsDatetime(approx=datetime(2000, 1, 1, tzinfo=timezone.utc), unix_number=True),
+            True,
+            id='unix-int',
```

Comment:
Devex feedback: it was annoying and confusing when the test failure just pointed to the line `assert value == dirty` and I had to find the param by name. I'd have been happier if the tests just said `assert 946684800 == IsDatetime(...)` etc. But I can see how it's also nice for test failures to be separate without having to define a test function for each case.

Reply:
I agree it's annoying, AFAIK it's just a limitation of pytest - I wouldn't want to write out every test manually.